client/src/components/CreateMeetup/CreateMeetup.js

```diff
@@ -69,6 +74,30 @@ function CreateMeetup() {
     "cooking",
   ];
 
+  useEffect(async () => {
+    if (!id || !user.id) {
+      return;
+    }
+    const { res, resJSON } = await fetcher.getMeetup(id);
+    if (!res.ok) {
+      return;
+    }
+    // only allow edit if user is the owner of the meetup
+    if (user.id !== resJSON.owner) {
+      return;
+    }
+    setIsEdit(true);
+    setTitle(resJSON.title);
+    setTime(new Date(resJSON.time));
+    setMeetupType(resJSON.location.url ? REMOTE : IN_PERSON);
+    // Physical locations cannot be prefilled, so have user refill manually for now
+    // setMeetupLocation(resJSON.location.name || null);
```

Comment:
Why can't we prefill it?

```suggestion
    setMeetupLocation({
      description: resJSON.location.name,
      coords: [resJSON.location.lon, resJSON.location.lat],
    });
```
```

Reply:
As I understand, LocationPicker.js takes in the raw location data that is transformed into lat/long/name before being stored in the DB. Prefilling the LocationPicker with coords/name doesn't work, and I'm not sure how to transform either coords or name back into raw location data.